generate/generate.go

```diff
@@ -344,31 +344,33 @@ func (g *Generator) fileOptions(pkg *loader.GunkPackage) (*desc.FileOptions, err
 		if f.Doc == nil {
 			continue
 		}
-		_, tag, err := loader.SplitGunkTag(g.fset, f.Doc)
-		if err != nil || tag == nil {
+		_, tags, err := loader.SplitGunkTag(g.fset, f.Doc)
+		if err != nil || len(tags) == 0 {
```

Comment:
I imagine this length check is unnecessary, since a range over an empty slice is a noop.

Reply:
Ah, completely missed this! Done.